### DISCUSSION TITLE
"Add Sample Data" information

### DISCUSSION THREAD

**Discussion by stevejburge** [roughsmoothengine/organize-series#735] [Feature Requests]
I wonder if it would help users if we had a "Add Sample Data" information

This would pre-populate 12 months of a magazine into a series. We could take the images and info from our demo site.

**Comment by olatechpro** [comment]
@stevejburge can you export the demo data for me or link the demo site so i can create the demo export from it

**Comment by stevejburge** [comment]
@htmgarcia can we help Ola with some good demo data for Series

I thinking a Magazine example:

- Series Group: Monthly Magazine
- Series: Year 1 and Year 2
- Posts: "January, Year 1" then "February, Year 2" etc

**Comment by stevejburge** [comment]
If we're moving towards shortcodes, maybe we also offer to create Pages with the key shortcodes?

**Comment by stevejburge** [comment]
Or how about this:

Series Category: Year 1 and Year 2
Series: January Year 1, January Year 2 etc

**Comment by htmgarcia** [comment]
@stevejburge @olatechpro I attach an XML with Series and Series Categories
https://drive.google.com/file/d/1d1R7rCMXKNIdomppGr8yMw2Ary5wqmHK/view?usp=sharing
